src/pages/TransactionPage/operations.generated.tsx

```diff
@@ -0,0 +1,187 @@
+import type * as Types from "../../api/types";
```

Comment:
@AlicanC I think we should ignore generated files on git. Any reason why it's better to keep it on history?

Reply:
It's a tech debt I carried from fuels-ts. It's not a problem there since we only have one generated file, but we can't do that here.

My plan is to ignore generated files, put `build-operations` in `postinstall` and add the schema to the repo. (At least the schema should be there to avoid people having to run fuel-core during `npm install`.)